### ISSUE TITLE
Plugin ImageExtra expects '?' on string everytime and images break

### ISSUE THREAD

**Issue by sjardim** [magabriel/trefoil#3]
Miguel, first of all, thank you for this great project that makes working on more advanced projects on easybook a piece of cake. 

The ImageExtra plugin checks for the presence of a '?' on the image string to apply the str_replace for 'images/', so if I just use  ![text](images/figure_5.8.png) it won't match and the build will show a warning:

`[WRN] Can't open input file: No such file or directory (/tmp/trefoil/easybook_pdf/images/images/figure_5.8.png)`

But if I use `![](images/figure_5.8.png?class="wide")` it will apply the str_replace and everything's good. 
 
An else on the preProcessImages function should resolve it. :) Would you like a PR?

**Comment by magabriel** [comment]
Yes, please, send a PR and I will look into it. And thanks for your comments.

**Comment by sjardim** [comment]
Great, I'll do a couple more tests and send it later.
